documentation/library_guide/api_for_sycl_kernels/random.rst

```diff
@@ -37,6 +38,8 @@ Type                                                                  Descriptio
 ``ranlux48_base``                                                     ``oneapi::dpl::subtract_with_carry_engine<std::uint64_t, 48, 5, 12>``
 ``ranlux24``                                                          ``oneapi::dpl::discard_block_engine<ranlux24_base, 223, 23>``
 ``ranlux48``                                                          ``oneapi::dpl::discard_block_engine<ranlux48_base, 389, 11>``
+``philox4x32``                                                        ``oneapi::dpl::philox_engine<uint_fast32_t, 32, 4, 10, 0xD2511F53, 0x9E3779B9, 0xCD9E8D57, 0xBB67AE85>``
```

Comment:
to check before the merge (I am not sure yet), if we align the order of consts with P2075R6 or wait till the corrections are published (where the multipliers should be swapped)

Reply:
need to apply: https://cplusplus.github.io/LWG/issue4134

Reply:
Applied, thanks for pointing this out.